site/docs/v1/tech/client-libraries/netcore.adoc

```diff
@@ -37,11 +37,101 @@ include::docs/v1/tech/client-libraries/_static-patch-note.adoc[]
 :client_library_language!:
 
 
+== FusionAuth Templates for .NET
+
+FusionAuth has a collection of .NET templates available as a NuGet package. Quickly create new secured .NET applications with FusionAuth templates using the .NET CLI or Visual Studio. The new applications are pre-configured to use FusionAuth for authentication and authorization. Only standard .NET security libraries are used in the templates.
+
+Currently, the following .NET templates are implemented:
+
+* FusionAuth Blazor Server Application.
+* FusionAuth MVC Application.
+* FusionAuth Web API Application.
+
+
+=== Requirements
+
+* A FusionAuth installation. You can link:/docs/v1/tech/installation-guide/[install FusionAuth locally, or sign up for a hosted account].
+* .NET Core 7.0 SDK or higher.
+* Visual Studio 2022 for Mac, version 17.6 and above (optional).
+* Visual Studio 2022 for Windows, version 17.6 and above (optional).
+
+
+=== Installing the Templates
+
+FusionAuth .NET templates are available on link:https://www.nuget.org/packages/FusionAuth.Templates/[NuGet]. You can install them by running the following command in your terminal.
+
+```bash
```

Comment:
The copy icon does not work for me here.

Reply:
for us the copy buttons never work when we serve the documentation locally or on our dev server, but everything works on production - not sure why. Can you confirm if the other copy buttons are working for you and it's just this one that is problematic?

Reply:
Copy buttons only work for me when I visit localhost:4000/localhost:3000, not 0.0.0.0:4000/0.0.0.0:3000. I suspect it is a JS loading issue with the library that provides the rating (based on js console output).

Reply:
That is the issue.